cloudify_rest_client/events.py

```diff
@@ -100,7 +100,8 @@ def delete(self, deployment_id, include_logs=False, message=None,
                                     deployment_id=deployment_id,
                                     **kwargs)
 
-        response = self.api.delete(uri, params=params)
+        response = self.api.delete(uri, params=params,
+                                   expected_status_code=200)
```

Comment:
Not 204?

Reply:
in events and executions, we do return the deleted count. Which is, as you know, allowed by the rfc.

Reply:
CY-3535 is about updating the rest docs to reflect all these changes